scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -39,26 +39,6 @@ build-linux-stable:
     - echo -n "v${RELEASE_VERSION}" > ./artifacts/BUILD_RELEASE_VERSION
     - cp -r scripts/* ./artifacts
 
-check-transaction-versions:
```

Comment:
Yea it is really annoying since it is broken, but will there be a fix or replacement for it?